requirements/base.in

```diff
@@ -1,10 +1,10 @@
 # Core requirements for using this application
 
-Flask-Dance[sqla]>=0.9,<0.10
```

Comment:
I would suggest to move all the pinned packages to `constraints.txt` and run make the upgrade. 

Once you do that, check if there is any failing test.